sdk/purview/azure-purview-sharing/azure/purview/sharing/_serialization.py

```diff
@@ -882,7 +882,9 @@ def serialize_iter(self, data, iter_type, div=None, **kwargs):
         """Serialize iterable.
 
         Supported kwargs:
+        # cspell:disable-next-line
```

Comment:
All your added content will be overwritten by codegen when next generation. So maybe you could do like: https://github.com/Azure/azure-sdk-for-python/blob/1f9192a3bc3880edf9b44cc23e1cd3914b8b9b0b/.vscode/cspell.json#L1337-L1341